cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -698,6 +698,11 @@ func (g *genConversion) doSlice(inType, outType *types.Type, sw *generator.Snipp
 				funcName := g.funcNameTmpl(inType.Elem, outType.Elem)
 				sw.Do(fmt.Sprintf("if err := %s(&(*in)[i], &(*out)[i], s); err != nil {\n", funcName), argsFromType(inType.Elem, outType.Elem))
 			} else {
+				// TODO: This triggers on v1.ObjectMeta <-> api.ObjectMeta and
+				// similar because neither package is the target package, and
+				// we really don't know which package will have the conversion
+				// function defined.  This fires on basically every object
+				// conversion outside of pkg/api/v1.
```

Comment:
yeah, it's pretty ugly and not easy to solve in the current design.  Gonna delete this comment for cleanliness